myapp/src/Day6/Que5/UserProfile.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Userprofile from './Userprofile';
+
+describe('Userprofile Component', () => {
+  it('renders user information correctly', () => {  
+
+    const userData = [
```

Comment:
@anantmishra-successive you are comparing data from 2 different sources. Tests should be written in such a way that we compare input and output, checking from the same source that they are same.